geometry_msgs/Pose initial_left_foot_position
geometry_msgs/Pose initial_right_foot_position
geometry_msgs/Pose initial_center_of_mass_position

geometry_msgs/Pose final_left_foot_position
geometry_msgs/Pose final_right_foot_position
geometry_msgs/Pose final_center_of_mass_position

bool start_with_left_foot

walk_msgs/Footprint2d[] footprints
---
walk_msgs/WalkPath path
